404.html

```diff
@@ -18,17 +18,17 @@
         <meta name="twitter:creator" content="@fossasia">
         <meta name="twitter:image" content="http://fossasia.github.io/fossasia15/img/fossasia-dark.png">
 
-        <link rel="shortcut icon" href="fossasia.ico" type="image/x-icon" />
-        <link href="css/bootstrap.min.css" rel="stylesheet" type="text/css" media="all"/>
-        <link href="css/flexslider.min.css" rel="stylesheet" type="text/css" media="all"/>
-        <link href="css/elegant-icons.min.css" rel="stylesheet" type="text/css" media="all"/>
-        <link href="css/font-awesome.min.css" rel="stylesheet" type="text/css" media="all"/>
-        <link href="css/pe-icon-7-stroke.min.css" rel="stylesheet" type="text/css" media="all"/>
-        <link href="css/lightbox.min.css" rel="stylesheet" type="text/css" media="all"/>
-        <link href="css/theme-lava.css" rel="stylesheet" type="text/css" media="all"/>
-        <link href="css/custom.css" rel="stylesheet" type="text/css" media="all"/>
-        <link href="css/custom-blog.css" rel="stylesheet" type="text/css" media="all"/>
-        <link href="css/bootstrap-theme.min.css" rel="stylesheet" type="text/css" media="all"/>
+        <link rel="shortcut icon" href="https://2019.fossasia.org/fossasia.ico" type="image/x-icon" />
```

Comment:
This isn't good way to give `href`

Reply:
Can you be clearer about the problem with this? If I changed such links to something like `/path`, as the bases of forks can differ in level (https://2019.fossasia.org, https://kualeyi.github.io/2019.fossasia.org), they would fail. If I changed them to something like `path`, they would be relative URLs with relative paths, obviously failing for nonexistent and non-working pages in subsubfolders etc.